.github/workflows/test.yml

```diff
@@ -1,4 +1,4 @@
-name: Test LocalGovDrupal Guides
+name: Test LocalGov Drupal
```

Comment:
Does the name of the test really need to change?

Reply:
Guess this should be Test LocalGov Drupal Guides?

@stephen-cox Do you want to confirm and I'll edit and merg.

Reply:
@ekes It should indeed be 'Test LocalGov Drupal Guides'. Happy for you to fix and merge. I'll do it later if you don't get around to it.

Reply:
Updated it in the github editor :)